version.js

```diff
@@ -151,16 +151,30 @@ function tryParseVersionBranch(branchName) {
 
 /**
  * Checks the PR to see if the major version should be incremented based on labels.
+ * @param {{ owner: string, repo: string }} repo
  * @param {string} commitMessage
- * @returns {Promise<'minor' | 'major'>}
+ * @returns {Promise<SemVer>}
  */
-async function getIncrementType(commitMessage) {
+async function getDefaultBranchNextVersion(repo, commitMessage) {
+  const previousRelease = await getLatestReleaseVersion(repo);
   const prNumber = tryParsePrNumber(commitMessage);
   if (prNumber === undefined) {
-    return "minor";
+    return previousRelease.inc("minor");
+  }
+  const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+  const pr = await octokit.rest.pulls.get({
+    ...repo,
+    pull_number: prNumber,
+  });
+  const prRef = pr.data?.head?.ref;
+  if (prRef !== undefined) {
+    const asVersion = tryParseVersionBranch(prRef);
```

Comment:
what does `asVersion` mean here?

Reply:
We're seeing if we can infer the branch name as a version - or return `undefined` if we can't. Have renamed to `prBranchVersion` instead.